packages/frontend/src/pages/mode-selection/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useIntl } from 'react-intl';
+import { Col, Row, Typography } from 'antd';
+
+import { FightMode } from './const';
+import { useModeSelection } from './hooks';
+import ModeCard from '../../components/ModeCard';
+import { SkywalkerImage, StarshipImage } from '../../assets/images';
+
+const ModeSelectionPage = () => {
+  const intl = useIntl();
+  const { onSelectMode } = useModeSelection();
+
+  return (
+    <>
+      <Typography.Title style={{ textAlign: 'center' }}>
+        {intl.formatMessage({
+          id: 'mode.select',
+          defaultMessage: 'Select mode',
+        })}
+      </Typography.Title>
+      <Row gutter={16}>
+        <Col span={24} md={12} style={{ display: 'flex' }}>
+          <ModeCard
+            testId={FightMode.PEOPLE}
+            img={SkywalkerImage}
+            onClick={() => onSelectMode(FightMode.PEOPLE)}
```

Comment:
i think the whole `useModeSelection` hook is overcomplicating simple things. Why not use here simply `() => navigate(ROUTES.people)`